semgrep/tests/e2e/snapshots/test_cli_test/test_cli_test_time/results.json

```diff
@@ -1,6 +1,10 @@
-targets/cli_test/basic/basic.py
-rule:rules.cli_test.basic.basic-test: Basic test
-2:print(1 == 1)
+
+ targets/cli_test/basic/basic.py 
```

Comment:
Any idea why these are highlighted?

Reply:
seems to me these should never have had the `.json` file suffix